src/core/thread/network_data.cpp

```diff
@@ -592,7 +592,7 @@ ThreadError NetworkData::SendServerDataNotification(bool aLocal, uint16_t aRloc1
     Message *message;
     Ip6::MessageInfo messageInfo;
 
-    mSocket.Open(&HandleUdpReceive, this);
+    mSocket.Open(&NetworkData::HandleUdpReceive, this);
```

Comment:
MSBUILD throws an error. It seems to support one of the following patterns (openthread uses both):

1. `mSocket.Open(HandleUdpReceive, this);`
2. `mSocket.Open(&NetworkData::HandleUdpReceive, this);`

Ideally, we should use the same syntax everywhere, but I'm not necessarily signing up for that. :)